analyzer/build.gradle

```diff
@@ -38,3 +43,21 @@ publish {
     description = 'Code generation for Java/Android database access.'
     website = 'https://github.com/novoda/sqlite-analyzer'
 }
+
+pluginBundle {
```

Comment:
@xrigau this is the publish plugin from the gradle portal (com.gradle.plugin-publish). We have used it for this plugin but then decided to go the "old route" via bintray. I wanted to leave it as an example. I have update the description.